supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -299,10 +299,25 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
   }
 
   private def resolveJobCommand(localizedInputs: CallInputs): Try[String] = {
-    if (runtimeAttributes.dockerImage.isDefined)
+    val command = if (runtimeAttributes.dockerImage.isDefined) {
       modifyCommandForDocker(call.task.instantiateCommand(localizedInputs, callEngineFunction, identity), localizedInputs)
-    else
+    } else {
       call.task.instantiateCommand(localizedInputs, callEngineFunction, identity)
+    }
+    command match {
```

Comment:
You could simply map the command instead of the redundant pattern match. 
```scala
command map { cmd => tellMetadata(Map("command" -> cmd )) }
```
Failure and logging of command instantiation is already handled elsewhere.